src/commands/publish.ts

```diff
@@ -103,11 +109,11 @@ const newMajorVersion = async (): Promise<void> => {
         );
         const newVersion =
             res.data.versions.reduce((accMax: any, item: any) => {
-                return item.isMajor && item.isActive
-                    ? Math.max(accMax, item.version)
-                    : accMax;
+                return item.isMajor ? Math.max(accMax, item.version) : accMax;
             }, 1) || 1;
```

Comment:
Suggestion: extract the reducer function to better describe what it does. 
```ts
        const getLatestVersion = (accMax: any, item: any) => {
                return item.isMajor ? Math.max(accMax, item.version) : accMax;
            }, 1);

        const newVersion = res.data.versions.reduce(getLatestVersion) || 1;
```

Reply:
Actually, I'm not even sure we even need the ternary statement here, since `Math.max(1, 1)` will return `1`.
So it can probably be simplified to this:
```ts
        const getLatestVersion = (currentHighest: any, item: any) => Math.max(currentHighest, item.version);
        const newVersion = res.data.versions.reduce(getLatestVersion, 1) || 1;
```

Reply:
Yes, but we have to check whether it's a major version though, but I guess the highest version would be the max just published.